scripts/screeps_rl_live_dashboard.py

```
@@ -0,0 +1,811 @@
+            return
+        self.write_json(HTTPStatus.NOT_FOUND, {"ok": False, "error": "not found"})
+
+    def do_POST(self) -> None:
+        parsed = urlparse(self.path)
+        if parsed.path != "/refresh":
+            self.write_json(HTTPStatus.NOT_FOUND, {"ok": False, "error": "not found"})
+            return
+        with self.server.refresh_lock:
+            try:
+                refresh = refresh_metrics(self.server.config.db_path, self.server.config.artifact_root)
+            except Exception as error:  # pragma: no cover - defensive handler boundary
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)